tests/conformance_ruby/filter_test.rs

```diff
@@ -161,11 +159,10 @@ fn test_sort_natural() {
 }
 
 #[test]
-#[should_panic] // liquid-rust#246
 fn test_compact() {
     let assigns = v!({
         "words": ["a", nil, "b", nil, "c"],
-        "hashes": [{ "a": "A" }, { "a": nil }, { "a": "C" }],
+        "hashes": [{ "a": "A" }, { "a": nil }, { "a": "C" }, {}],
```

Comment:
Was the `{}` added to test another case?

`conformance_ruby` is a direct port of tests from the ruby implementation and I'd prefer to stay aligned with them and instead to add our own tests to [`filters.rs`](https://github.com/cobalt-org/liquid-rust/blob/master/tests/filters.rs) so I can more easily audit for changes in the ruby tests

Reply:
I thought `compact` should handle missing attributes, but on a second read of the documentation they don't mention this.

We can go with either:
* Undefined behaviour - Don't test this at all
* Error - Check for this in the filter, and add a test in another file.
* Drop such elements - Move the test to `filters.rs`.

I'm good with either way (although I think the last one is best for compatibility). What do you think?

Reply:
If they don't mention it and don't test it, then I'm not too worried which route we go with and am fine going with whatever you recommend.

Reply:
I took option 3 and moved the test to `filters.rs`.